install/kubernetes/helm/istio/values-istio-gateways.yaml

```diff
@@ -13,7 +13,8 @@ global:
   # configmap in those.
   omitSidecarInjectorConfigMap: true
 
-  # istio control plane namespace
+  # Istio control plane namespace: modify this if you installed the
+  # control plane in a different namespace than istio-system.
```

Comment:
I don't remember why this is needed - can you add an explanation ? Normally we get this from Release.Namespace.

Reply:
Oh, this is needed to tell the 2nd gateway where the control plane is.  I'll clarify it.